acceptance/suites/tests/00_smoke/master_starts_if_agent_run_before_ssl_files_inited.rb

```diff
@@ -36,8 +38,10 @@
 
 end
 
-step 'Disable facts reporting to PuppetDB while we munge certs' do
-  on(master, puppet("config set route_file /tmp/nonexistant.yaml"))
+step 'Disable reporting to PuppetDB while we munge certs' do
+  on(master, puppet("config set --section master route_file /tmp/nonexistant.yaml"))
```

Comment:
Is this because we can't delete settings via config set?

Reply:
I think they're working on fixing that... :)

Reply:
Ish, here it's because PDB installs a `route_file` in the default location, so we need to override the default here. The explicitly setting it to the default in the teardown is because we can't delete settings.

Alternatively, we could move and move back the route_file but I think we need to twizzle the other settings so seemed easier to do all the changes in settings.

Reply:
If we're looking at this line can we correct the spelling? "nonexistant -> nonexistent"

Reply:
of course

Reply:
fixed